evap/results/exporters.py

```diff
@@ -175,11 +184,43 @@ def export(self, response, course_types_list, include_not_enough_voters=False, i
                 percentage_participants = int((evaluation.num_voters / evaluation.num_participants) * 100) if evaluation.num_participants > 0 else 0
                 writec(self, "{}%".format(percentage_participants), "evaluation_rate")
 
+            if course_results_exist:
+                writen(self)
+                for evaluation, results in evaluations_with_results:
+                    if evaluation.course.evaluations.count() > 1:
+                        self.write_empty_cell_with_borders()
+                    else:
+                        self.write_empty_cell()
+
+                writen(self, _("Evaluation weight"), "bold")
+                for evaluation, results in evaluations_with_results:
+                    if evaluation.course.evaluations.count() > 1:
+                        writec(self, "{}%".format(evaluation.weight_percentage), "evaluation_weight")
+                    else:
+                        self.write_empty_cell()
+
+                writen(self, _("Course Grade"), "bold")
+                for evaluation, results in evaluations_with_results:
+                    if evaluation.course.evaluations.count() > 1:
+                        writec(self, evaluation.course.avg_grade, self.grade_to_style(evaluation.course.avg_grade))
+                    else:
+                        self.write_empty_cell()
+
+                writen(self)
+                for evaluation, results in evaluations_with_results:
+                    if evaluation.course.evaluations.count() > 1:
```

Comment:
that query is done five times for each evaluation. you can leave that if you want, i mean it's the export and we don't really care how fast it is, but if it does get too slow I won't be the one optimizing queries again.